src/window/vulkan/src/lib.rs

```diff
@@ -185,13 +235,13 @@ pub fn init<T: gfx_core::format::RenderFormat>(title: &str, width: u32, height:
         imageColorSpace: vk::COLOR_SPACE_SRGB_NONLINEAR_KHR,
         imageExtent: vk::Extent2D { width: width, height: height },
         imageArrayLayers: 1,
-        imageUsage: vk::IMAGE_USAGE_COLOR_ATTACHMENT_BIT | vk::IMAGE_USAGE_DEPTH_STENCIL_ATTACHMENT_BIT,
+        imageUsage: vk::IMAGE_USAGE_COLOR_ATTACHMENT_BIT,
         imageSharingMode: vk::SHARING_MODE_EXCLUSIVE,
         queueFamilyIndexCount: 1,
         pQueueFamilyIndices: &0,
         preTransform: vk::SURFACE_TRANSFORM_IDENTITY_BIT_KHR,
         compositeAlpha: vk::COMPOSITE_ALPHA_OPAQUE_BIT_KHR,
-        presentMode: vk::PRESENT_MODE_FIFO_RELAXED_KHR,
+        presentMode: vk::PRESENT_MODE_FIFO_KHR, // required to be supported
```

Comment:
Only `PRESENT_MODE_FIFO_KHR` is required to be supported following the specs. Checking the queried present modes for better options (ie `PRESENT_MODE_MAILBOX_KHR`) could be done.